src/styles.css

```diff
@@ -0,0 +1,91 @@
+body {
+    margin: 0px;
+}
+
+#worldmap {
+    width: 95%;
+
+
+}
+
+#earth{
+    font-family: "Brush Script MT";
+    font-size: 150px;
+    display: flex;
+    justify-content: center;
+    background-image: linear-gradient(to left, violet, indigo, blue, green, yellow, orange, red);
+    -webkit-background-clip: text;
+    -webkit-text-fill-color: transparent;
+    border: 0.5rem outset pink;
+    outline: 0.5rem solid khaki;
+    box-shadow: 0 0 0 2rem forestgreen;
+    border-radius: 12px;
+    margin: 2rem;
+    padding: 1rem;
+    outline-offset: 0.5rem;
+
+
+}
+
+#countries-title {
+    font-family: "Brush Script MT";
```

Comment:
Hier mist ook de default font. Aangezien dit het enige font is dat je oproept had je dit ook in de body kunnen zetten.